### DISCUSSION TITLE
is there a way to set number of cells in my drone model?

### DISCUSSION THREAD

**Discussion by koolakoff** [EdgeTX/edgetx#3396] [Q&A]
Good day, team,

I've got a BattAnalog widget at my EdgeTX 2.7.1 (tx16s ii)
And this widget auto-detects number of cells. And once I've got my 2 cells (high voltage) inserted - it detects it as 'almost-out-of-charge-3-cell-battary'. 

Is there any way to specify the batt sells number for the widget?

Thanks in advance
Peter

**Comment by 3DRacer** [comment]
in the flight controller SW, e.g. Betaflight, the cell voltages must be set correctly so that the number of cells is correctly recognized.
